packages/mobile/src/send/SendConfirmation.test.tsx

```diff
@@ -124,42 +123,47 @@ describe('SendConfirmation', () => {
     await flushMicrotasksQueue()
 
     const feeComponent = getByTestId('feeDrawer/SendConfirmation/totalFee/value')
-    expect(getElementText(feeComponent)).toEqual('$0.0466')
+    expect(getElementText(feeComponent)).toEqual('$0.0266')
 
-    // NOTE: CELO fees are currently not combined into the total.
-    // TODO: This should equal more than $1.33, depending on the CELO fee value.
+    // Subtotal is $1.33, which is added the fee amount.
     const totalComponent = getByTestId('TotalLineItem/Total')
-    expect(getElementText(totalComponent)).toEqual('$1.33')
+    expect(getElementText(totalComponent)).toEqual('$1.36')
   })
 
-  it('shows a generic `calculateFeeFailed` error when fee estimate fails due to an unknown error', async () => {
```

Comment:
Do you think showing `---` is enough for users to understand there was an error?

Reply:
Not really, no. When fee estimation fails the user also gets an error thing on the top saying it failed, but that happens in the previous screen so it's hard for them to make the connection. I'm just following how it used to work, but I agree we should probably improve on this, I'll create an issue

Reply:
Here's the issue: https://app.zenhub.com/workspaces/retention-squad-sprint-board-612f2ef98ef2b800116e6043/issues/valora-inc/wallet/1549